presets/4.3/tune/Tehllama_GTB339_3in_SWPR-Spec_2S-3S_Race_TUNE_Options.txt

```diff
@@ -0,0 +1,173 @@
+#$ TITLE: 3in Lightweight Race GTB339 3S/2S Tune
+#$ FIRMWARE_VERSION: 4.3
+#$ CATEGORY: TUNE
+#$ STATUS: EXPERIMENTAL
+#$ KEYWORDS: GTB339, Toothpick, Spec, Race, Llama, SWPR, MultiVoltage
+#$ AUTHOR: Daniel Appel / Tehllama
+#$ DESCRIPTION: This tune is intended for GTB339 3in 1105-5500KV Racers on on bidir DShot ESCs at 48kHz PWM with MedHigh timing. 
+#$ DESCRIPTION: These fly best on 2S-3S batteries in the 450-650mAh capacity range, resulting in an AUW to 95-105g on these motors.  Works with biblade and triblade props (e.g. Gemfan 3016/3018, HQ 3x1.8x3/3x1.5x3, etc.)
+#$ DISCLAIMER: This tune will auto-select profiles based on battery voltage at plugin for 2S and 3S packs.
+#$ WARNING: Requires 12-Pole Motors!!! Use at your own risk.  Extensive testing has been done across 37+ builds, however not every craft will run best on this tune.  13A ESCs may experience failures on 3S batteries.
+#$ DISCUSSION: https://app.slack.com/client/T221J1F6J/C0228HK2MD4/
+
+#$ INCLUDE: presets/4.3/tune/defaults.txt
+
+
+# master
+set dshot_bidir = ON
+set motor_poles = 12
+
+set dshot_idle_value = 475
+# Fallback if dynamic idle does not work or is disabled
+
+set thrust_linear = 20
+
+# profile 3:  3S
+profile 2
+
+set vbat_sag_compensation = 77
+set anti_gravity_gain = 4400
+set iterm_rotation = ON
+set iterm_relax = RPY
+set iterm_relax_cutoff = 33
+set yaw_lowpass_hz = 125
+set p_pitch = 54
+set i_pitch = 72
+set d_pitch = 44
+set f_pitch = 99
+set p_roll = 49
+set i_roll = 66
+set f_roll = 90
+set p_yaw = 49
+set i_yaw = 66
+set f_yaw = 90
+set d_min_pitch = 33
+set d_max_advance = 0
+set auto_profile_cell_count = 3
+set feedforward_boost = 8
+set throttle_boost = 5
+set simplified_i_gain = 75
+set simplified_pi_gain = 110
+set simplified_feedforward_gain = 75
+set simplified_pitch_d_gain = 110
+set simplified_pitch_pi_gain = 110
+simplified_tuning apply
+
+# profile 1: 2S
+profile 1
+
+set vbat_sag_compensation = 77
+set anti_gravity_gain = 4400
+set iterm_rotation = ON
+set iterm_relax = RPY
+set iterm_relax_cutoff = 37
+set yaw_lowpass_hz = 125
+set p_pitch = 82
```

Comment:
all these direct PID values are going to be overwritten by simplified_tuning apply, so you don't have to mention them.